doc/code/qml_debugging.rst

```diff
@@ -1,11 +1,8 @@
 qml.debugging
 =============
+.. currentmodule:: pennylane.debugging
 
-.. automodapi:: pennylane.debugging
-    :no-heading:
-    :no-inherited-members:
-    :skip: PLDB
-    :skip: pldb_device_manager
+.. automodule:: pennylane.debugging
```

Comment:
Note that the table is now no longer [showing up](https://xanaduai-pennylane--5894.com.readthedocs.build/en/5894/code/qml_debugging.html):
![image](https://github.com/PennyLaneAI/pennylane/assets/49409390/814461cc-8e59-462a-8dd0-c86959b08552)

I still think it's a useful table.

Reply:
Added 👍🏼